### PR TITLE
refactor: remove accidental awk and add ci-check for it

### PR DESCRIPTION
**Pull request by port19x**
# Pull Request Template

## Type of change

- [x] Bug fix
- [ ] Feature
- [ ] Documentation update

## Description

I briefly verified that the feature still works with naruto (finished), one piece (2 days) and boruto (multiple results)

### CODE REVIEW

**Comment by port19x** [comment]
Very well, it fails as it should

**Review by justchokingaround — COMMENTED** [review]
id still prefer one sed over 2 greps and 2 cuts

**Comment by port19x** [comment]
The second grep and the two cuts could be optimized away, but I don't think the gains are too noticable.
How would that sed look like?

File: ani-cli [hunk 1]
```diff
@@ -471,6 +446,11 @@ case "$search" in
             [ -z "$query" ] && query=$(printf "" | external_menu "" "Search anime: ")
             [ -z "$query" ] && exit 1
         fi
+        # for checking new releases by specifying anime name
+        if [ "$search" = "nextep" ]; then
+            time_until_next_ep "$query"
+            exit 0
+        fi
```
Comment (port19x) [inline ani-cli:453]:
```suggestion
        [ "$search" = "nextep" ] && time_until_next_ep "$query"
```

File: ani-cli [hunk 2]
```diff
@@ -200,27 +200,14 @@ search_anime() {
...
-
-    exit 0
+    curl -s -G "$animeschedule/api/v3/anime" --data-urlencode "q=$1" | sed 's|"id"|\n|g' | sed -nE 's|.*,"route":"([^"]*)","premier.*|\1|p' | while read -r anime; do
+        data=$(curl -s "$animeschedule/anime/$anime" | sed '1,/"anime-header-list-buttons-wrapper"/d' | sed -nE 's|.*countdown-time-raw" datetime="([^"]*)">.*|Next Raw Release: \1|p;s|.*countdown-time" datetime="([^"]*)">.*|Next Sub Release: \1|p;s|.*english-title">([^<]*)<.*|English Title: \1|p;s|.*main-title".*>([^<]*)<.*|Japanese Title: \1|p')
+        status="Ongoing"
+        color="33"
+        printf "%s\n" "$data"
+        ! (printf "%s\n" "$data" | grep -q "Next Raw Release:") && status="Finished" && color="32"
+        printf "Status:  \033[1;%sm%s\033[0m\n---\n" "$color" "$status"
+    done
```
Comment (port19x) [inline ani-cli:210]:
```suggestion
    done
    exit 0
```

**Review by port19x — COMMENTED** [review]
Might as well properly move everything into `time_until_next_ep`

**Comment by port19x** [comment]
`git diff 63b7b`

**Review by 71zenith — APPROVED** [review]

**Pull request merged**